Makefrag

```diff
@@ -19,8 +19,9 @@ JVM_MEMORY ?= 2G
 SBT ?= java -Xmx$(JVM_MEMORY) -Xss8M -XX:MaxPermSize=256M -jar $(base_dir)/sbt-launch.jar
 SHELL := /bin/bash
 
+ROCKET_CLASSES ?= "$(base_dir)/target/scala-2.11/classes:$(base_dir)/chisel3/target/scala-2.11/*"
```

Comment:
I don't think it's necessary for this PR but I think we should consider moving to an `sbt-pack` approach: https://github.com/xerial/sbt-pack